packages/auth/README.md

```diff
@@ -2,11 +2,13 @@
 
 ## 📚 About
 
+This module handles auth in your applications and other ttoss modules.
+
 This module is intended to use with AWS Cognito. It uses [AWS Amplify](https://docs.amplify.aws/lib/auth/getting-started/q/platform/js) under the hood.
 
 [Amplify Auth configuration](https://docs.amplify.aws/lib/auth/start/q/platform/js#re-use-existing-authentication-resource) must be provided in your App to make Auth Module works properly.
 
-## 🚀 Get Started
+## 🚀 Getting Started
```

Comment:
Here we have to tell users that they need to install `@ttoss/notifications` and wrap `NotificationsProvider` to auth module works properly.